scripts/timelock-delay-change/index.ts

```diff
@@ -0,0 +1,267 @@
+#!/usr/bin/env ts-node
+
+import { execSync } from 'child_process';
+import * as readline from 'readline';
+
+interface DelayChangeOptions {
+  network: string;
+}
+
+class TimelockDelayChanger {
```

Comment:
Just a thought, no need to change it now: There are quite a lot functions that might be reusable between the different scripts, along with the index.sh maybe could be generalized?